lib/include/tick/prox/prox_with_groups.h

```diff
@@ -100,6 +101,30 @@ class TProxWithGroups : public TProx<T> {
     this->blocks_length = blocks_length;
     is_synchronized = false;
   }
+
+ protected:
+  template <class Archive>
+  void serialize(Archive &ar) {
+    ar(cereal::make_nvp("Prox", cereal::base_class<TProx<T> >(this)));
+    ar(CEREAL_NVP(positive));
```

Comment:
`TProxWithGroups` should not have its own `positive` and should be using `TProx` one!

Reply:
Yeah I thought that was a bit strange

Reply:
DO we fix this here or in another issue?

Reply:
https://github.com/X-DataInitiative/tick/issues/213